### PR TITLE
some function add try_catch/离线发送任务修改和离线发送任务调用 接口 新增

### PR DESCRIPTION
**Pull request by nwljy**
some function add try_catch
离线发送任务
离线发送任务调用 接口 新增

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,21 +1,32 @@
...
+from flask import Flask , request , render_template , send_file,make_response,jsonify
 from tinydb import Query , TinyDB
-import io,json
+import io,json,os
 import pic_email as wc
-
+import jobs as jobs
+import traceback
 app = Flask(__name__)
+defaultResult_0="KEY is not right"
```
Comment (zhaofeng-shu33) [inline app.py]: Where is the documentation?

File: pic_email.py [hunk 2]
```diff
@@ -30,18 +30,31 @@ def send_email(email):  # 将result.png 发送到指定的 邮件
...
-    font_number = ImageFont.truetype('font/1.ttf', size=35)  # 编号的字体和字号
-    imwidth, imheight = im.size
-    font_width, font_height = draw.textsize(name, font_name)  # 获取名字的大小
-    draw.text(((imwidth - font_width - font_name.getoffset(name)[0]) / 2, 470), text=name, font=font_name,
-              fill=(0, 0, 0))  # 写上名字 x使用了居中
-    draw.text(xy=(310, 285), text=number, font=font_number)  # 写上编号
-    im.save("result.png")
-    send_email(email)  # 可直接发送到邮箱
+    try:
+
```
Comment (zhaofeng-shu33) [inline pic_email.py]: no empty lines here

File: app.py [hunk 3]
```diff
@@ -1,21 +1,32 @@
...
 # 2: Make Image& Send email sucess
 # 3: Have been used
 # 4:
 # 5:Send wrong
-def return_msg(message):
-    if type(message) is dict:
-        message = json.dumps(message)
-    return message
+def errorHanddler(e=Exception,isOut=True):
+    print(str(e))
```
Comment (zhaofeng-shu33) [inline app.py]: all print lines of debug information should be deleted, or use some `verbose` technique to switch it on or off.

File: jobs.py [hunk 4]
```diff
@@ -1,26 +1,40 @@
...
             to_email=user.email,
             subject='快来领取您的wuhan2020证书',
-            content='感谢您的辛苦付出，请点击链接领取您的志愿证书\n',
+            content='<p>感谢您的辛苦付出，请点击链接领取您的志愿证书\n',
         )
-        update_status(email=user.email, status=1)
-        time.sleep(random.random() * 10)
+    update_status(email=user.email, status=1)
+    
+def send_notice_email():
```
Comment (zhaofeng-shu33) [inline jobs.py]: when this function will be called?

**Comment by lovepoem** [comment]
怎么ci失败了

**Comment by zhaofeng-shu33** [comment]
two many changes.

**Pull request closed without merging**